src/leap/mail/imap/mailbox.py

```diff
@@ -514,13 +514,14 @@ def _get_imap_msg(messages):
                 d_imapmsg = []
                 for msg in messages:
                     d_imapmsg.append(getimapmsg(msg))
-                return defer.gatherResults(d_imapmsg)
+                return defer.gatherResults(d_imapmsg, consumeErrors=True)
 
             def _zip_msgid(imap_messages):
                 zipped = zip(
                     list(msg_range), imap_messages)
                 return (item for item in zipped)
 
+            # XXX not called??
```

Comment:
no, the code should be called, but I didn't have the time to focus on where on this round.